client/scala/scala-armada-client/src/test/scala/ArmadaClientSuite.scala

```diff
@@ -213,6 +248,79 @@ class ArmadaClientSuite extends munit.FunSuite {
     ac.deleteQueue(qName)
   }
 
+  test("ArmadaClient.cancelJobs()") {
+    val ac = ArmadaClient("localhost", testPort)
+    val qName = "nonexistent-queue-" + Random.alphanumeric.take(8).mkString
+    var jobs = Seq[String]()
+
+    ac.createQueue(qName)
+
+    // Submit 3 jobs
+    for (i <- 1 to 3) {
+      val response = ac.submitJobs(qName, "testJobSetId", List(new JobSubmitRequestItem()))
+      assertEquals(response.jobResponseItems.length, 1)
+      jobs = jobs :+ response.jobResponseItems(0).jobId
+    }
+
+    // Required for processing Futures in test
+    implicit val ec: scala.concurrent.ExecutionContext = scala.concurrent.ExecutionContext.global
+
+    // Cancel the first two of the jobs and validate they are cancelled
+    val cancelRes = ac.cancelJobs(new JobCancelRequest().withJobIds(Seq[String](jobs(0), jobs(1))))
```

Comment:
I think this can be:
```
    val cancelRes = ac.cancelJobs(new JobCancelRequest().withJobIds(jobs.take(2))
```

Reply:
Thanks - I didn't know Scala Sequences had that method - fixed in commit ffdc1e18.